library/core/src/main/java/com/google/android/exoplayer2/Format.java

```diff
@@ -48,7 +48,18 @@
   public final @Nullable String id;
   /** The human readable label, or null if unknown or not applicable. */
   public final @Nullable String label;
-
+  /** Track selection flags. **/
+  @C.SelectionFlags
+  public final int selectionFlags;
+  /** Track role descriptor value, or {@link C#ROLE_UNSET} if unknown or not applicable. **/
+  @C.Role
+  public final int role;
```

Comment:
And having a single `roleFlags`, rather than separate `role` and `accessibility` fields.